app/src/androidTest/java/com/github/epfl/meili/MainActivityTest.kt

```diff
@@ -98,15 +73,6 @@ class MainActivityTest {
         Intents.intended(toPackage("com.github.epfl.meili"))
     }
 
-    @Test
```

Comment:
Why is this test being removed?

Reply:
I forgot to add it back after some debugging, good catch!